phonon/update.py

```diff
@@ -89,7 +90,10 @@ def __cache(self):
             cached = json.loads(self.__process.client.get(self.resource_id) or "{}")
             self.merge(cached)
         self.cache()
-        self.ref.increment_times_modified()
+        if not self.init_cache:
+            # If caching on init, we do not increment times_modified to prevent
```

Comment:
@akellehe Yeah, after giving this more thought, I don't even think this approach holds up in the case of a single process.  (ie if another Update comes in for the same resource, then we want it to merge with the original update before it caches).  I am, however, starting to think that the init_cache functionality should be pulled out and handled by the LruCache.  Things are caching even before they are set on the LruCache, which is probably something we don't want. For example, in the case described above, the __cache function is doing its own merging, and then the lru_cache runs a merge as well. (ie Update2 merges with Update1 because of the init_cache, so now update Update 2 has data from 2+1, and then the lru_cache merges Update2 into Update1 again, so we are left with the data for Update 1+2+1).  Going to close this pull request and explore some other options.